### DISCUSSION TITLE
CHanging log levels when running a DAG

### DISCUSSION THREAD

**Discussion by waardd** [apache/airflow#31341] [Q&A]
In a DAG we use

```
import logging
TASKLOGGER = "airflow.task"
task_logger = logging.getLogger(TASKLOGGER)

task_logger.info("INFO message")
task_logger.debug("DEBUG message")
```

This results in the info message being visible in the log but the debug not.
By default this is fine but my question is how can we see the debug messages without changing the airflow configuration?

For example with a parameter when starting the DAG manual

**Comment by potiuk** [comment]
Just look at standard python logging API. You will find it by searching standard python docs. They also have examples how to set logging levels. You can set the log level of any logger/handler as you wish. And you can add a parameter in yor dag to control it. Just look for Params in airflow docs
